node/libs/roles/src/validator/messages/consensus.rs

```diff
@@ -169,10 +172,53 @@ pub struct LeaderCommit {
     pub justification: CommitQC,
 }
 
+/// Utility for incrementally constructing a `PrepareQC` instance .
+pub struct PrepareQCBuilder {
+    instance: PrepareQC,
+    val_set_size: usize,
```

Comment:
I suggest we actually keep an Arc<ValidatorSet> in the builder.

Reply:
I ended up removing the builder altogether.